json_api_drupal_filters/__init__.py

```diff
@@ -1 +1,2 @@
-from .filter_parser import FilterParser, FilterError
+from .filter_parser import FilterParser
+from .filter_errors import *
```

Comment:
```suggestion
from .filter_errors import FilterError, RootKeyUsedError, NoGroupOrCondition
```
Star import could hinder tools like pylint, pyflakes or sonarcloud to detect bugs.